app/controllers/ui_job_wizard_controller.rb

```diff
@@ -24,6 +24,19 @@ def map_template_inputs(template_inputs_with_foreign)
     template_inputs_with_foreign.map { |input| input.attributes.merge({:resource_type_tableize => input.resource_type&.tableize }) }
   end
 
+  def default_category
+    default_template&.job_category
+  end
+
+  def default_template
+    setting_value = Setting['remote_execution_form_job_template']
+    return nil unless setting_value
```

Comment:
`return nil` is equivalent to just `return`, there's no need to be this explicit